fe/fe-core/src/test/java/com/starrocks/server/SharedDataStorageVolumeMgrTest.java

```diff
@@ -784,4 +784,17 @@ public void testCreateHDFS() throws DdlException, AlreadyExistsException {
         svm.updateStorageVolume("test", storageParams, Optional.of(false), "");
         Assert.assertEquals(false, svm.getStorageVolumeByName(svName).getEnabled());
     }
+
+    @Test
+    public void testCreateStorageVolumeWithInvalidParams() throws DdlException {
+        String svName = "test";
+        StorageVolumeMgr svm = new SharedDataStorageVolumeMgr();
+        List<String> locations = Arrays.asList("{hdfs://abc}");
+        Map<String, String> storageParams = new HashMap<>();
+        Assert.assertThrows(DdlException.class,
+                () -> svm.createStorageVolume(svName, "hdfs", locations, storageParams, Optional.empty(), ""));
+
+        Assert.assertThrows(DdlException.class,
+                () -> svm.createStorageVolume(svName, "abc", locations, storageParams, Optional.empty(), ""));
```

Comment:
add s3 and azblob case coverage as well, make sure all the types are covered and checked.